harness/testAtomics.js

```diff
@@ -44,6 +39,11 @@ function testWithAtomicsInBoundsIndices(f) {
     var good_indices = [
         (view) => 0/-1, // -0
         (view) => '-0',
+        (view) => undefined, // 0
+        (view) => Number.NaN, // 0
+        (view) => 3.5,
+        (view) => '3.5',
```

Comment:
if the length is 3 or more this will fail.

`0.5` and `'0.5'` should avoid this